python-bareos/bareos/bsock/lowlevel.py

```diff
@@ -236,12 +236,12 @@ def __connect(self):
         return auth
 
     def __connect_plain(self):
-        self.socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         # initialize
         try:
+            timeout = 30
             if self.timeout:
-                self.socket.settimeout(self.timeout)
-            self.socket.connect((self.address, self.port))
+                timeout = self.timeout
```

Comment:
What about
```
timeout = self.timeout or 30
```
?
I am not sure if its more pythonic or not

Reply:
Yeah maybe we should simply set the self.timeout in init, then adapt it if configured, and apply the resulting value.
which would be certainly more cleaner and easier to understand.